olin-build/abe-web
# Upgrade to React 17

## Description

* Upgrades from React 15 to React 17
* Requires Node v10
* Bumps a couple of other old dependencies
* Tracks a `yarn.lock` file so installed package versions are mirrored across machines

## Testing

1. Checkout the branch
2. Change your version of Node to v10 by running `nvm use node 10`
3. Run `yarn` to install the new dependency versions
4. Run `yarn dev` to launch the app

Make sure it compiles and runs fine.
